website/architecture/bundled-hermes.md

```diff
@@ -0,0 +1,156 @@
+---
+id: bundled-hermes
+title: Bundled Hermes
+---
+
+This page gives an overview of **how** Hermes and React Native **are built**.
+
+If you're looking into instructions on how to use Hermes in your app, you can find instructions on this other page: [using Hermes](/docs/hermes)
+
+:::caution
+Please note that this page serves as a technical deep dive and is targeted for users which are building extensions on top of Hermes or React Native. General users of React Native should not need to know in-depth information on how React Native and Hermes interact.
+:::
+
+## What is 'Bundled Hermes'
+
+Starting with React Native 0.69.0, every version of React Native will be **built alongside** to a Hermes version. We call this distribution model **Bundled Hermes**. 
+
+From 0.69 on, you will always have a JS engine that has been built and tested alongside each React Native version that you can use.
+
+## Why we moved to 'Bundled Hermes'
+
+Historically, React Native and Hermes followed two **distinct release processes** with distinct versioning. Having distinct releases with distinct numbers created confusion in the OSS ecosystem, where it was not clear if a specific version of Hermes was compatible with a specific version of React Native (i.e. you needed to know that Hermes 0.11.0 was compatible only with React Native 0.68.0, etc.)
+
+Both Hermes and React Native, share the JSI code ([Hermes here](https://github.com/facebook/hermes/tree/main/API/jsi/jsi) and [React Native here](https://github.com/facebook/react-native/tree/main/ReactCommon/jsi/jsi)). If the two JSI copies of JSI gets out of sync, a build of Hermes won't be compatible with a build of React Native. You can read more about this problem, [ABI incompatibility here](https://github.com/react-native-community/discussions-and-proposals/issues/257).
+
+To overcome this problem, we've extended the React Native release process to download and build Hermes, and make sure only one copy of JSI is used when building Hermes.
+
+Thanks to this, we can release a version of Hermes whenever we release a version of React Native, and be sure that the Hermes engine we built is **fully compatible** with the React Native version we're releasing. We're shipping this version of Hermes alongside the React Native version we're doing, hence the name _Bundled Hermes_.
+
+## How this will impact app developer
+
+As mentioned in the introduction, if you're a app developer this change **should not affect** you directly.
+
+The following paragraphs describe which changes we did under the hood and explains some of the rationales, for the sake of transparency.
+
+### iOS Users
+
+On iOS, we've move the `hermes-engine` you're using.
+
+Prior to React Native 0.69, users would download a remote podspec ([defined here](https://github.com/CocoaPods/Specs/blob/master/Specs/5/d/0/hermes-engine/0.11.0/hermes-engine.podspec.json)).
```

Comment:
```suggestion
Prior to React Native 0.69, users would download a pod (here you can find the [podspec](https://github.com/CocoaPods/Specs/blob/master/Specs/5/d/0/hermes-engine/0.11.0/hermes-engine.podspec.json)).
```
The `hermes-engine` 0.11 is a fully fledged pod, published also in the official cocoapods website. When we say "download a remote podspec" we could introduce some confusion because people may think that it was a custom solution.